src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateService.kt

```diff
@@ -21,17 +20,16 @@ class ReleaseDateService(
     return maxNumberOfWorkingDaysToUpdateLicenceTimeOutStatus.workingDaysAfter(LocalDate.now(now ?: clock))
   }
 
-  fun isInHardStopPeriod(licence: Licence, overrideClock: Clock? = null): Boolean {
+  fun isInHardStopPeriod(sentenceDateHolder: SentenceDateHolder, overrideClock: Clock? = null): Boolean {
     val now = overrideClock ?: clock
-    val hardStopDate = getHardStopDate(licence)
+    val hardStopDate = getHardStopDate(sentenceDateHolder)
     val today = LocalDate.now(now)
 
     if (hardStopDate == null) {
-      log.warn("Licence with id: ${licence.id} has no CRD")
```

Comment:
this wasn't really actionable and was relying on the implementation details of an inner method